src/function-builder.ts

```diff
@@ -127,28 +164,38 @@ export class FunctionBuilder {
    * functions.region('us-east1', 'us-central1')
    */
   region(...regions: Array<typeof SUPPORTED_REGIONS[number]>): FunctionBuilder {
-    if (assertRegionsAreValid(regions)) {
-      this.options.regions = regions;
-      return this;
-    }
+    assertRegionsValidity(regions);
+
```

Comment:
Not sure if we need all these line breaks, although I do see your intent to separate the validity check, the main action in the function, and the return.

Reply:
Yes, exactly. Hope that's fine like this, but I can change it if needed.

Reply:
Should be fine this way